app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training.md

```diff
@@ -0,0 +1,110 @@
+
+It's important to check the requirements for each course you apply to, so you do not waste time on an application you’re unlikely to be successful with. 
+
+### Your qualifications 
+
+All postgraduate teacher training courses require you to have: 
+
+* a degree in any subject 
+* GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary) 
+
+Most course providers will accept a degree grade of 2:2 or higher, but some may require you to have a 2:1. 
+
+Check each course provider’s requirements on their course page. You can also [find out more about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach). 
+
+### Your right to work in the UK 
+
+You’ll need the right to work or study in the UK to do your teacher training in England. 
+
+Some training providers offer courses with visa sponsor available. You can filter by ‘visa sponsorship’ to [find courses where visas can be sponsored](https://www.find-postgraduate-teacher-training.service.gov.uk/).   
+
+[Find out more about applying for your visa to train to teach in England](/non-uk-teachers/visas-for-non-uk-trainees).
+
+## The qualifications the course awards 
+
+Most schools in England need you to have [qualified teacher status (QTS)](/what-is-qts) to teach. 
+
+Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/what-is-a-pgce), and some will award both. 
+
+Without QTS, you may be limiting the schools you’ll be able to teach in and will not receive the same benefits within your training or career. 
+
+## If the course comes with a salary
+
+### Fee-funded teacher training
+
+Most teacher training courses are fee-funded – this means you have to pay tuition fees and will not earn a salary while you train. 
+
+There are ways to fund your training, for example, through tuition fee and maintenance loans and scholarships and bursaries. 
+
+You can [find out more about the different ways to fund your training](/funding-and-support).
+
+### Salaried teacher training
+
+Some courses are salaried, which means you’ll receive an unqualified teacher’s salary while you train. 
+
+Places on salaried courses are limited and very competitive. You can [find out more about what you’ll need to get on a salaried teacher training course](/funding-and-support/salaried-teacher-training). 
+
+## The course provider 
+
+Your teacher training course might be provided by: 
+
+* a university (sometimes referred to as university-led training) 
+* a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 
+* [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)  
+
+Once you’ve found a course that you’re interested in, you could talk to the training provider before you apply. For example, you might want to find out: 
+
+* where your placements and theoretical learning might take place 
```

Comment:
Slightly repetitive on `place` - is there any alternative? But very minor - happy for it to stay as is!